apps/root/src/common/utils/parsing.ts

```diff
@@ -400,12 +400,16 @@ export const parseTokensForPicker = ({
   balances: TokenBalances;
   customTokens?: TokenList;
 }) => {
-  const tokenKeys = Object.keys(tokenList);
-  const customTokenKeys = Object.keys(customTokens || {});
+  const mergedTokenLists = {
+    ...tokenList,
+    ...customTokens,
```

Comment:
but custom tokens can be undefined D:

Reply:
btw I think the order should be reversed, so we replace any custom token with the one that we have on our list that should be more curated

Reply:
Absolutely! I didn't considered this case, thank you